### PR TITLE
Made the usage of pre-existing nsg more user friendly

### PR DESCRIPTION
**Pull request by pabowers**
The nsg-id is now really difficult to find in the portal.  This should make using a custom nsg much easier for the user

### CODE REVIEW

File: deploy/vm/modules/nsg_for_hana/variables.tf [hunk 1]
```diff
@@ -1,8 +1,12 @@
 variable "az_region" {}
 
+variable "nsg_name" {
+  description = "The name of the nsg group to be created."
```
Comment (tniek) [inline deploy/vm/modules/nsg_for_hana/variables.tf]: Nit: "NSG", "nsg group"

File: deploy/vm/modules/nsg_for_hana/variables.tf [hunk 2]
```diff
@@ -13,10 +17,6 @@ variable "sap_instancenum" {
   description = "The sap instance number which is in range 00-99"
 }
 
-variable "sap_sid" {
-  default = "PV1"
-}
-
```
Comment (tniek) [inline deploy/vm/modules/nsg_for_hana/variables.tf]: I believe this is an unrelated change?

Comment (pabowers) [reply deploy/vm/modules/nsg_for_hana/variables.tf]: As I changed the nsg for hana variables to take a name of the nsg and the `sap_sid` was only needed for the name of the nsg, I thought that since it was no longer needed, it made sense to remove it in this PR
